src/docs/sphinx/tutorials/step11/Tutorial.rst

```diff
@@ -0,0 +1,234 @@
+
+
+Initial and Boundary Conditions: defining properties with the FieldSpecifications
+-----------------------------------------------------------
+
+The next step is to specify fields, including:
+
+  - The initial value (the in-situ stresses and traction at the wellbore wall have to be initialized)
+  - The boundary conditions (the reduction of wellbore pressure and constraints of the outer boundaries have to be set)
+
+In this tutorial, we need to specify isotropic horizontal stress (:math:`\Sigma_h` = -11.25 MPa) and vertical stress (:math:`\Sigma_v` = -15.0 MPa). 
+To reach equilibrium, a compressive traction :math:`P_w` = -11.25 MPa is instantaneously applied at the wellbore wall ``rneg`` at time :math:`t` = 0 s, which will then be gradually reduced to a lower value (-2.0 MPa) to let wellbore contract.
+The remaining parts of the outer boundaries are subjected to roller constraints.  
+These boundary conditions are set up through the ``FieldSpecifications`` section.
+
+
+.. literalinclude:: ../../../../../examples/plasticity/DruckerPrager_Verification.xml
+    :language: xml
+    :start-after: <!-- SPHINX_DP_WELLBORE_BC -->
+    :end-before: <!-- SPHINX_DP_WELLBORE_BC_END -->
+
+
+With ``tractionType="1"``, traction is applied to the wellbore wall ``rneg`` as a pressure specified from the product of scale ``scale="-11.25e6"`` and the outward face normal. 
+A table function ``timeFunction`` is used to define the time-dependent traction ``ExternalLoad``. 
+The ``coordinates`` and ``values`` form a time-magnitude
+pair for the loading time history. In this case, the loading magnitude decreases linearly as the time evolves. 
+
+.. literalinclude:: ../../../../../examples/plasticity/DruckerPrager_Verification.xml
+    :language: xml
+    :start-after: <!-- SPHINX_DP_WELLBORE_TABLE -->
+    :end-before: <!-- SPHINX_DP_WELLBORE_TABLE_END -->
+
+You may note :
+
+ - All initial value fields must have ``initialCondition`` field set to ``1``;
+ - The ``setName`` field points to the previously defined box to apply the fields;
+ - ``nodeManager`` and ``faceManager`` in the ``objectPath`` indicate that the boundary conditions are applied to the element nodes and faces, respectively;
+ - ``fieldName`` is the name of the field registered in GEOSX;
+ - Component ``0``, ``1``, and ``2`` refer to the x, y, and z direction, respectively;
+ - And the non-zero values given by ``Scale`` indicate the magnitude of the loading; 
+ - Some shorthand, such as ``xneg`` and ``xpos``, are used as the locations where the boundary conditions are applied in the computational domain. For instance, ``xneg`` means the portion of the computational domain located at the left-most in the x-axis, while ``xpos`` refers to the portion located at the right-most area in the x-axis. Similar shorthand include ``ypos``, ``yneg``, ``zpos``, and ``zneg``;
+ - The traction loading has a negative value due to the surface sign convention in GEOSX, where tensile stress is considered positive.
```

Comment:
It hard to understand this sentense because "traction loading hase a negative value" sounds contradict with "tensile stress is considered positive"